briskedu/brisk-learning
# Handle Cancel button on login

Summary:
Handling when user clicks Cancel in login flow

https://app.asana.com/0/1203963408726580/1204219627247632

Test Plan:
1. Confirm cancel closes window
2. Confirm login works
